torch_xla/_internal/pjrt.py

```diff
@@ -111,6 +118,11 @@ def initialize_multiprocess(local_rank: int, local_world_size: int):
     tpu.configure_topology(local_rank, local_world_size)
   elif runtime.device_type() == 'NEURON':
     neuron.initialize_env(local_rank)
+  elif runtime.device_type() == 'GPU':
+    global_world_size = xu.getenv_as('WORLD_SIZE', int)
+    global_rank = get_global_rank(local_rank)
```

Comment:
Also for consideration: `xu.getenv_as(xenv.RANK, int, local_rank)`

Reply:
This is handy. Thanks!
I replaced it with a function `should_initialize_dist_runtime` per a previous comment.